### PR TITLE
[xray] Deprecation of  rabbitmq-ha chart

### PR DESCRIPTION
**Pull request by chukka**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by eldada — APPROVED** [review]

**Comment by eldada** [comment]
We need to provide instructions on how to do the migration. Since we are replacing RMQ, it needs to be with a graceful shutdown of the system.

**Review by danielezer — APPROVED** [review]

File: stable/xray/CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ All changes to this chart will be documented in this file.
 
 ## [6.0.0] - Sep 2, 2020
 * **Breaking change:** Changed `imagePullSecrets` value from string to list.
-* **Breaking change:** Added `image.registry` and `common.xrayVersion` is changed to `image.tag` under analysis,indexer,persist,server and router sections
+* **Breaking change:** Added `image.registry` and `image.version` is changed to `image.tag` under analysis,indexer,persist,server and router sections
```
Comment (chukka) [inline stable/xray/CHANGELOG.md:13]: @rahulsadanandan  please don't change previous version's changelog , Add changes in current changelog

Comment (rahulsadanandan) [reply stable/xray/CHANGELOG.md]: Its changed now.

**Comment by peters95** [comment]
This will break openshift.

**Review by peters95 — CHANGES_REQUESTED** [review]
Can we explain why we are doing this? 

This will adversely impact our integrations for Partner engineering like Openshift which uses the Rabbitmq HA chart for our Enterprise customers.

**Comment by TJM** [comment]
NOTE: The bitnami chart *also* supports HA, just set replicaCount: 3 ... We just need to get rid of the deprecated and broken rabbitmq-ha chart *name* .. its only a very minor edit to switch over, and that can be handled by a release note.  See #1270

**Comment by chukka** [comment]
#1355

**Pull request closed without merging**